xrtpy/image_correction/tests/test_remove_lightleak.py

```diff
@@ -46,7 +47,7 @@ def get_composite_data_files():
 
 
 @pytest.mark.parametrize(("idlfile", "compfile"), data_files)
-def test_lightleak(idlfile, compfile, allclose):
```

Comment:
This is going to conflict with #284, mostly because I've not just removed allclose but also heavily refactored the tests.

Reply:
To prevent that I can undo these changes.